.idea/misc.xml

```diff
@@ -1,6 +1,15 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
-  <component name="ProjectRootManager" version="2" languageLevel="JDK_11" default="true" project-jdk-name="1.8" project-jdk-type="JavaSDK">
+  <component name="DesignSurface">
+    <option name="filePathToZoomLevelMap">
+      <map>
+        <entry key="..\:/Users/lukas/AndroidStudioProjects/Workout_Planer/app/src/main/res/layout/activity_main.xml" value="0.2" />
+        <entry key="..\:/Users/lukas/AndroidStudioProjects/Workout_Planer/app/src/main/res/layout/activity_navigation.xml" value="0.2" />
+        <entry key="..\:/Users/lukas/AndroidStudioProjects/Workout_Planer/app/src/main/res/layout/activity_navigation2.xml" value="0.25416666666666665" />
```

Comment:
Again, an entry to `navigation2.xml`. I do not think it should be here. Check this file with a bit more attention, since I do not know what should be included here. This one is your call